### ISSUE TITLE
urllib3, six, and installation issue

### ISSUE THREAD

**Issue by kkemink** [marxanweb/general#271]
I'm currently working with Windows 10 and tried the latest full installation of marxanweb. I keep running into issues with the python libraries though. I get a warning when I first install the app that the urllib3 is not the appropriate version and is incompatible, and that the six library is not the appropriate version either. Then although the install will continue, the web app won't open but spits out an error message about the six library instead. I've tried updating the packages a couple different times and ways but it seems to break some sort of vital connection in the structure because it only breeds more error messages. Has anyone else come across this issue, and if so was there a solution you could share?

**Comment by andrewcottam** [comment]
Installation of Python packages on Windows can be problematic, but it should be straightforward for Marxan Web because it uses the [Conda](https://docs.conda.io/en/latest/#) and installs all of the dependencies in a self-contained conda environment. If you are having problems installing on Windows you can either use the hosted service [here](https://app.marxanweb.org) (which requires no installation) or we can try to identify the problem. How familiar are you with conda and Python package management? BTW - whats your name :)

**Comment by kkemink** [comment]
Hey, yes - sorry I'm Kaylan. Thanks for your reply. It'd be great if we could try to id the problem but I'm not an expert in Python/conda - just enough to be dangerous. My guess is that the problem starts with the warning pasted in below, which occurs during installation. 

![image](https://user-images.githubusercontent.com/52049313/97187587-9516af80-1770-11eb-8bf9-2073eb14bca5.png)

After that, the program installs, but when you try to open it you get the following errors:

![image](https://user-images.githubusercontent.com/52049313/97187850-e161ef80-1770-11eb-8d5c-c39fce8625b8.png)

Not sure if that helps to clarify?

**Comment by andrewcottam** [comment]
Yes any screenshots help thanks Kaylan. A few questions to start with:
- Have you installed any other software that uses Python on your computer? e.g. QGIS, ArcGIS etc.
- If you go to the cmd line and type the following what does it say:
```
conda info --envs
```
- If you type python at the command line, what happens?

**Comment by kkemink** [comment]
Yes - Python 2.7 is installed as I have to use it for ArcMap, which, unfortunately doesn't use any later versions of Python. 

I get the following output:

C:\Users\kkemi>conda info --envs
# conda environments:
#
                         C:\Users\kkemi\Anaconda2
base                  *  C:\Users\kkemi\AppData\Local\MarxanWeb\Miniconda3


C:\Users\kkemi>

**Comment by andrewcottam** [comment]
Thanks - can you type this in the cmd prompt and copy and paste the results back here thanks:
```
conda activate base
conda list
```

**Comment by kkemink** [comment]
![image](https://user-images.githubusercontent.com/52049313/97220149-08341c00-1799-11eb-854a-f73414bef04a.png)

![image](https://user-images.githubusercontent.com/52049313/97220088-f2bef200-1798-11eb-9c4f-d023a8fc0228.png)

![image](https://user-images.githubusercontent.com/52049313/97220046-dfac2200-1798-11eb-8306-26e6cb6565b9.png)
